### ISSUE TITLE
seems not supporting react 15+

### ISSUE THREAD

**Issue by zoubingwu** [EnixCoda/ReactLifeHook#1]
```jsx
import React from "react";
import { render } from "react-dom";
import { activate } from "react-lifecycle-hooks";

activate({ React, compat: "legacy" });

class ClassDemo extends React.Component {
  render() {
    return <div>Hello World</div>;
  }
}

render(<ClassDemo />, document.getElementById("demo"));
```

more details see https://codesandbox.io/s/8lxvlql49j

a peerDependencies key is suggested in package.json to indicate the required react version

**Comment by EnixCoda** [comment]
Sorry for me ignoring those projects still on React before v16.0.
A patch is on the way~

**Comment by EnixCoda** [comment]
🎉Patch has been released! (v0.2.5)

https://codesandbox.io/s/72rqvrpv0q
